### ISSUE TITLE
Carry over on opening (not only creating) current's day file

### ISSUE THREAD

**Issue by bitozoid** [bastibe/org-journal#372]
**Is your feature request related to a problem? Please describe.**
I like to create TODOs in the future. That is already implemented by `org-journal-new-scheduled-entry`. However, when I reach that day, the file is already created and carry over does not work.

**Describe the solution you'd like**
I'd like carry over to be called every time I open (not only when I create) the current's day file.

**Describe alternatives you've considered**
An explicit interactive function would also work to force carry over.

**Comment by bastibe** [comment]
Good idea! It might also be useful to flash a little warning that the previous day contains un-migrated TODO items.

Would you like to try implement a solution in a pull request?

**Comment by bitozoid** [comment]
I don't know enough about emacs lisp and emacs. Plus, I'm short on time.

**Comment by bastibe** [comment]
If anyone else would like to contribute this as a pull request, I'd be grateful.
